tsdapiclient/session.py

```diff
@@ -0,0 +1,73 @@
+
+import os
+import time
+
+from datetime import datetime, timedelta
+
+import yaml
+
+from tsdapiclient.tools import (
+    check_if_key_has_expired, check_if_exp_is_within_range, debug_step
+)
+
+HOME = os.path.expanduser('~')
```

Comment:
I was just thinking we should be using $XDG_CONFIG_HOME/tacl/session for the session store with fallback to $HOME/.config/tacl/session if this doesn't complicate things on Windows. Perhaps there is a built-in mechanism in Python to get the default config store which should be somewhere in their %AppData% hierarchy. I can have a look at it later.

Reply:
great!, I have already merged, but I guess another PR would do :)